### DISCUSSION TITLE
Clean Code Practice

### DISCUSSION THREAD

**Discussion by kirtimandwade20** [NeoSOFT-Technologies/rest-java-spring-boot#18] [Ideas]
Clean architecture : project should be organized in a manner so that it’s easy to understand and easy to change as the project grows. 

https://medium.com/javarevisited/clean-java-code-79b6da1878c9
https://www.baeldung.com/spring-boot-clean-architecture
https://medium.com/swlh/clean-architecture-java-spring-fea51e26e00
https://github.com/damienbeaufils/spring-boot-clean-architecture-demo

**Comment by YashC2410** [comment]
Should be surely added in Milestone 2 if possible
